README.md

```diff
@@ -629,13 +649,24 @@ php:
   visualdiff:
 ```
 
-## Peer Review Guidelines for Automated Updates
+## Contributor Docs
+
+This repo is public.
 
-These are guidelines for conducting peer reviews on automated dependency update pull requests created by Renovate. 
+Please be careful to remove sensitive customer specifics when posting Issues or comments.
 
-## Automated Testing with GitHub Actions
+First time contributors need a maintainers approval for automated tests to run.
+(This is so we aren't at risk of getting a big CI bill accidentally, or maliciously.)
 
-### Overview
+Peer Reviewing by looking at PR code changes is nice.
+
+Testing PR code changes on real sites is extra beneficial.
+
+### Peer Review Guidelines for Automated Updates
+
+These are guidelines for conducting peer reviews on automated dependency update pull requests created by Renovate.
+
+#### Overview
```

Comment:
Do we need an overview section here? Could it not just be a part of this section above with only one sentence?

Reply:
cc @mrdavidburns just trying to figure out the best way to merge this new section with the work that was merged in https://github.com/Lullabot/drainpipe/pull/411